finagle-memcached/src/main/scala/com/twitter/finagle/Memcached.scala

```diff
@@ -58,13 +57,18 @@ private[finagle] object MemcachedTraceInitializer {
                   Trace.recordBinary(key, "Hit")
                   misses.remove(key)
                 }
-                misses foreach { Trace.recordBinary(_, "Miss") }
+                misses foreach {
+                  Trace.recordBinary(_, "Miss")
```

Comment:
pulling this trace call into a function literal shaves an allocation on the request path